ggsdk/sdk.py

```diff
@@ -38,3 +38,12 @@ def save_dataset(self,
             exists (default=False)
         """
         self._core.save_dataset(data_generator, dataset_id, overwrite)
+
+    def promote_model(self, model_name: str, nlp_task: str, environment: str):
+        """Call promote model api.
+
+        :param model_name: Name of the model to promote within cloud storage
```

Comment:
@HarryDunham If this is the right way to document user-facing methods could you change the save_dataset call to use this as well? It's using `@param:` instead.

Reply:
@alexiudice Done! This docstring format is [reStructuredText](https://en.wikipedia.org/wiki/ReStructuredText), which seems far more ubiquitous than the previous Epytext format which is what you used. Additionally, all the other python projects use reStructuredText so it makes sense to align everything there. Additionally, if you want this parameter docstring scheme to be autogenerated within Pycharm you can go to File -> Settings -> Python Integrated Tools -> Docstrings section and set `Docstring format: reStructuredText` which'll autopopulate the docstring with the methods arguments.

Reply:
Thanks for this!